src/profile.ts

```diff
@@ -181,31 +210,45 @@ export default class Profile {
   minLayerHeight: number[];
   minPrintSpeed: number[];
   minSkirtLength = 4;
+  mmuSegmentedRegionInterlockingDepth = 0;
   mmuSegmentedRegionMaxWidth = 0;
+  multiMaterialPurging = 140;
   notes = '';
   nozzleDiameter: number[];
+  nozzleHighFlow: boolean[];
+  onlyOnePerimeterFirstLayer = false;
   onlyRetractWhenCrossingPerimeters = false;
   oozePrevention = false;
+  originOffset: [number, number, number] = [0, 0, 0];
   outputFilenameFormat =
     '{input_filename_base}_{layer_height}mm_{filament_type[0]}_{printer_model}_{print_time}.gcode';
+  overHangFanSpeed0: number[];
+  overHangFanSpeed1: number[];
+  overHangFanSpeed2: number[];
+  overHangFanSpeed3: number[];
   overhangs = true; // 'Detect Bridging Perimeters'
+  overHangSpeed0 = 15;
+  overHangSpeed1 = 15;
+  overHangSpeed2 = 20;
+  overHangSpeed3 = 25;
```

Comment:
I am not sure what you mean by this

Reply:
In `index.ts` we set `bridgeSpeed` based on project settings. Rather than leaving these default values, should we match current behaviour by setting these all to that same value?